utils/logger/logrus_logger.go

```diff
@@ -48,6 +48,7 @@ func SetupGlobal() {
 			return
 		},
 	})
+	logrus.SetLevel(logrus.DebugLevel)
```

Comment:
I think we should handle `PMM_DEBUG` / `PMM_TRACE` there

Reply:
Yes, but I set it for now to have the same level in zap and in logrus.

Reply:
Actually, the default value should be info – and on that level we should log too much, and should not log sensitive details. Both `PMM_DEBUG` and `PMM_TRACE` should enable debug level (and development mode in zap); logrus trace level should not be used